### PR TITLE
[Doc Improvement][Dev comm support links][Add links][2541127]

### PR DESCRIPTION
**Pull request by v-ajiths**
Modified the metadata for better SEO results and added a cross link in the "See also" section.

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/debug.md](https://github.com/MicrosoftDocs/msteams-docs/blob/84eb634fbfa455b1402d210deb129912509d5ae1/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fdebug.md) | 91 | 10 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/799f7382-e515-4b47-8885-23fad7e700c1) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/m365-apps/overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/84eb634fbfa455b1402d210deb129912509d5ae1/msteams-platform%2Fm365-apps%2Foverview.md) | 93 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f1060618-98ff-4de3-a0b7-2432579a0389) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by v-habibaumme — DISMISSED** [review]
Approved

File: msteams-platform/m365-apps/overview.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Extend Teams apps across Microsoft 365 (preview)
-description: In this module, know how to build and update your Teams app experiences across other high-usage areas of Microsoft 365. 
+description: In this article, learn how to build, update, and extend your Teams app experiences and how to use the Teams app manifest across other high-usage areas of Microsoft 365.
```
Comment (ashishguptaiitb) [inline msteams-platform/m365-apps/overview.md:3]: We are not using the app manifest across various surfaces. This edit is incorrect. 
End-users are using apps across surfaces. Manifest just helps create such apps.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/debug.md](https://github.com/MicrosoftDocs/msteams-docs/blob/585548765fa1992f7f38c7223e51e363e44818f6/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fdebug.md) | 91 | 10 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/3a65b5d9-7c86-4f28-a414-3ca9a2c788d4) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/m365-apps/overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/585548765fa1992f7f38c7223e51e363e44818f6/msteams-platform%2Fm365-apps%2Foverview.md) | 93 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/4a113460-c4dc-43c8-a2b6-0b72f0b511e8) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/debug.md](https://github.com/MicrosoftDocs/msteams-docs/blob/e038f989c8b337df6aa3af7d4e2a34970ab2003a/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fdebug.md) | 91 | 10 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/8c52655a-e840-4f88-adcf-2228d157f5fe) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/m365-apps/overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/e038f989c8b337df6aa3af7d4e2a34970ab2003a/msteams-platform%2Fm365-apps%2Foverview.md) | 93 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/78839994-c390-4f44-b34d-9bea821e29ff) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/m365-apps/overview.md (hunk 1 above)
Comment (v-ajiths) [reply msteams-platform/m365-apps/overview.md]: Edited and updated the description.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/debug.md](https://github.com/MicrosoftDocs/msteams-docs/blob/4c5b44c5b977a4958b78d47f3bbcead57cf88bba/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fdebug.md) | 91 | 10 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/eff319bd-b168-4cbd-92ca-4304ad4b0c7b) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/m365-apps/overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/4c5b44c5b977a4958b78d47f3bbcead57cf88bba/msteams-platform%2Fm365-apps%2Foverview.md) | 93 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a9663e85-aa85-4466-900f-73a5f658e1e7) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by ashishguptaiitb — APPROVED** [review]
LGTM after edits.

**Pull request merged**